cql3/statements/select_statement.cc

```diff
@@ -1069,6 +1069,10 @@ indexed_table_select_statement::do_execute(query_processor& qp,
         // Obviously, if there are no clustering columns, then we can work at
         // the granularity of whole partitions.
         whole_partitions = true;
+    } else if (_schema->get_column_definition(to_bytes(_index.target_column()))->is_static()) {
```

Comment:
side rant, not relevant to this series: we need to migrate everything to bytes_view and string_view finally, to avoid needless instantiations like this